CHANGELOG.md

```diff
@@ -2,464 +2,311 @@
-     - Zone enablement
-- Rest APIs
-  - Tag Collection /api/tags 
-  - Tag Management (assign and unassign to/from resources)
-  - Foundational
-     - Virtual attribute support  
-     - Id/Href separation
-  - Policy Management
-     - Query policy and policy profiles conditions
- - VM Management
-     - Custom Attributes
-     - Add LifeCycle Events
-- I18n Status
+#### I18n
   - All strings in the views have been converted to use gettext (I18n) calls
   - Can add/update I18n files with translations
-- Service Dialogs: Added Dynamic checkbox
-- Orchestration
-  - Provisioning dialog generator
-  - Enabled Reporting / Tagging
-  - Automate service models
-
-### [Changed](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+19+Ending+Feb+16%2C+2015%22+label%3Aenhancement)
-
-- UI
-  - Login screen converted to Bootstrap / Patternfly
-  - Header, navigation, and outer layouts converted to Bootstrap / Patternfly
-  - Advanced search converted to Bootstrap / Patternfly
-  - Stacks screens have icons now
-  - Orchestration Insight
-  - Schedule editor converted to AngularJS
-  - UI Customizations with Less
-  - Dashboard tabs updated
-
-### [Removed](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+19+Ending+Feb+16%2C+2015%22+label%3A%22technical+debt%22)
-
-- I18N: All views converted to HAML
-- Removed Prototype
-
-### [Fixed](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+19+Ending+Feb+16%2C+2015%22+label%3Abug)
-
-- Notable fixes include:
- - EventEx is now disabled by default to help prevent event storms
- - Fixed ftp log collection regression
- - Fixed “High CPU usage” due to continually restarting workers, when a provider is unreachable or password is invalid
- - Fixed bug causing fleecing timeout
- - Fixed timeout issue with remove_from_disk method on a VM in Automate
- - Fixed duplicate VM name generation issue during provisioning
-
-## Unreleased - as of Sprint 18 end 2015-01-26
-
-### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+18+Ending+Jan+26%2C+2015%22+label%3Aenhancement)
-
-- Providers
-  - Added refresh status and errors, viewable on Provider Summary Page.
-  - Amazon EC2: Added C4 instance types.
-- Rest API: VM start, stop, suspend, delete actions added.
-- UI
- - Multi-character set language support
- - Cloud Stacks summary and list views
-- Service Dialogs: Dynamic field support for text boxes and text area boxes
-- IPv6: Ensure an IPv6 literal address, such as "[::1]", passed to the Excon gem removes the square brackets [], when establishing a Socket connection to the address, "::1".
 
-### [Changed](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+18+Ending+Jan+26%2C+2015%22+label%3Aenhancement)
+#### Service Dialogs
+- Dynamic field support for text boxes and text area boxes
+- Dynamic checkbox
+- Dynamic radio button support
+- Dynamic list refactor
+- Read only fields
+- Dynamic Dialog Date/Time Control
```

Comment:
"Read only fields" should be at the bottom of the list after all the Dynamic fields are listed.  The Dynamic fields could 
be listed in a single entry.
```
- Dynamic field support for: text box, text area, checkbox, radio button and Date/Time control.
- Dynamic list field refactored into standard drop-down field
- Read-only field support
```